params/upgrade_config_test.go

```diff
@@ -0,0 +1,272 @@
+// (c) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package params
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ava-labs/subnet-evm/precompile"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVerifyUpgradeConfig(t *testing.T) {
+	admins := []common.Address{{1}}
+	chainConfig := *TestChainConfig
+	chainConfig.TxAllowListConfig = precompile.NewTxAllowListConfig(big.NewInt(1), admins)
+
+	type test struct {
+		upgrades            []PrecompileUpgrade
+		expectedErrorString string
+	}
+
+	tests := map[string]test{
+		"upgrade bytes conflicts with genesis (re-enable without disable)": {
+			expectedErrorString: "disable should be [true]",
+			upgrades: []PrecompileUpgrade{
+				{
+					TxAllowListConfig: precompile.NewTxAllowListConfig(big.NewInt(1), admins),
+				},
+			},
```

Comment:
can we add a case where disable is true, but the timestamps are the same as well?